gluon/dal.py

```diff
@@ -117,3 +117,11 @@ def _default_validators(db, field):
         DRIVERS['pg8000'] = pg8000
     except:
         pass
+if not DRIVERS.get('sqlite3'):
```

Comment:
Haha, yup, that's what fixed it!

On Tue, Apr 19, 2016 at 3:10 AM, Sina Hamedian <notifications@github.com>
wrote:

> In gluon/dal.py
> <https://github.com/slugiot/slugiot-server/pull/9#discussion_r60205104>:
>
> > @@ -117,3 +117,11 @@ def _default_validators(db, field):
> >          DRIVERS['pg8000'] = pg8000
> >      except:
> >          pass
> > +if not DRIVERS.get('sqlite3'):
>
> Wait, was this supposed to stop that sqlite error when you launch web2py
> initially?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/slugiot/slugiot-server/pull/9/files/5b32ac427eaa2622e9b349034214fc2a814633aa#r60205104>
>